Client/Application/LeanplumIntegration.swift

```diff
@@ -186,6 +199,18 @@ class LeanPlumClient {
         self.enabled = enabled
         Leanplum.setTestModeEnabled(!enabled)
     }
+    
+    func getEnabled() -> Bool {
+        // Certain services like FxA prepush dialogs need to know that Leanplum has started and can run A/B tests
+        return isLPEnabled()
+    }
+    
+    func isFxAPrePushEnabled() -> Bool {
+        if AppConstants.MOZ_FXA_LEANPLUM_AB_PUSH_TEST {
+            return self.useFxAPrePush
```

Comment:
If `self.useFxAPrePush` becomes an `LPVar`, this can change into:

```
return AppConstants.MOZ_FXA_LEANPLUM_AB_PUSH_TEST && useFxAPrePush?.boolValue()
```